src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.sql

```diff
@@ -893,6 +895,37 @@ AS
         THROW 50404, 'Instance not found', 1;
     END
 
+    -- Deleting indexed instance tags
```

Comment:
do you want to add some integration test to cover this?

Reply:
I need STOW checked in to add integration tests - otherwise I find that I have to build a lot of infrastructure around manually adding rows into the individual custom tag tables.

Reply:
sure, jsut make sure having a task tracking that.

Reply:
Changes look good now. Lets add integration tests and test it before checkin